### PR TITLE
Fix hard-coded admin notification email address

### LINKED ISSUE #950: Make admin triggers an email with details to core@beyondfog.com
Great work. However, when an admin is created, an email is sent to core@beyondfog.com with this information. Not ideal. :)

Putting that email address into process.env would solve this - not sure what you want to call it, otherwise I'd do it myself as a PR.

The literal email address is set in [lib/email.js @ line 164](https://github.com/Strider-CD/strider/blob/master/lib/email.js#L164).

### PR DESCRIPTION
**Pull request by oliversalzburg**
Fixes #950

### CODE REVIEW

**Comment by oliversalzburg** [comment]
Should probably be merged and published ASAP, but could be considered a breaking change :P

File: lib/users.js [hunk 1]
```diff
@@ -19,11 +19,21 @@ function makeAdmin(user, done) {
...
+        });
+      });
     }
 
     done(null, num);
   });
 }
+
+function getAdmins(done) {
+  User.find({ account_level: 1 }, function (err, admins) {
```
Comment (knownasilya) [inline lib/users.js]: Would this also send an email to the newly created user (if it's an admin)? Seems like that user should be ignored.

**Comment by oliversalzburg** [comment]
Should be better now

**Comment by knownasilya** [comment]
I think it's time to cut a release 👍

**Pull request merged**